CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ set (VERSION "${libplctag_VERSION_MAJOR}.${libplctag_VERSION_MINOR}.${libplctag_
 
 set (LIB_NAME_SUFFIX "${libplctag_VERSION_MAJOR}.${libplctag_VERSION_MINOR}.${libplctag_VERSION_PATCH}")
 
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 2.8...3.19)
```

Comment:
I am not sure about this one.   Why do you want to limit the version of CMake to <= 3.19?   Is there a breaking change at that point?